### PR TITLE
fix(topic-public): add styles to link

### PR DESCRIPTION
**Pull request by Slowlydev**
I made the link on the "ask" & "vote" pages more visiable

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -75,6 +76,7 @@
 	--text-primary: #ffffff;
 	--text-secondary: #888888;
 	--link-primary: #6464ff;
+	--link-background: #0a1c3f;
```
Comment (simylein) [inline src/index.css:79]: If it is always the same value as background-topic-1 why not use that instead?

File: src/index.css [hunk 2]
```diff
@@ -2,6 +2,7 @@
 	--text-primary: #000000;
 	--text-secondary: #808080;
 	--link-primary: #0000ff;
+	--link-background: #193469;
```
Comment (simylein) [inline src/index.css:5]: If it is always the same value as background-topic-1 why not use that instead?

File: src/index.css (hunk 1 above)
Comment (Slowlydev) [reply src/index.css:79]: Well I thought u like a new variable that describes the usage of that color

Comment (simylein) [reply src/index.css:79]: Please no

Comment (Slowlydev) [reply src/index.css:79]: Okay

**Pull request merged**